ui/src/main/java/ucar/nc2/ui/simplegeom/SimpleGeomUI.java

```diff
@@ -0,0 +1,897 @@
+/*
+ * Copyright (c) 1998-2018 University Corporation for Atmospheric Research/Unidata
+ * See LICENSE for license information.
+ */
+package ucar.nc2.ui.simplegeom;
+
+import thredds.client.catalog.ServiceType;
+import thredds.client.catalog.Dataset;
+import thredds.client.catalog.tools.DataFactory;
+import ucar.nc2.constants.FeatureType;
+import ucar.nc2.dataset.CoordinateAxis1D;
+import ucar.nc2.dataset.NetcdfDataset;
+import ucar.nc2.dt.GridCoordSystem;
+import ucar.nc2.dt.GridDataset;
+import ucar.nc2.dt.GridDatatype;
+import ucar.nc2.ui.geoloc.NavigatedPanel;
+import ucar.nc2.ui.geoloc.ProjectionManager;
+import ucar.nc2.ui.gis.MapBean;
+import ucar.nc2.ui.grid.*;
+import ucar.nc2.ui.util.Renderer;
+import ucar.nc2.ui.widget.*;
+import ucar.nc2.ui.widget.PopupMenu;
+import ucar.nc2.ui.widget.ProgressMonitor;
+import ucar.nc2.util.NamedObject;
+import ucar.unidata.geoloc.ProjectionImpl;
+import ucar.util.prefs.PreferencesExt;
+
+import javax.swing.*;
+import javax.swing.border.EtchedBorder;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeListener;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Formatter;
+import java.util.List;
+
+/**
+ * This is the thredds Data Viewer application User Interface for Simple Geometries.
+ *
+ * @author skaymen
+ */
+public class SimpleGeomUI extends JPanel {
+  static private final String DATASET_URL = "DatasetURL";
+  static private final String GEOTIFF_FILECHOOSER_DEFAULTDIR = "geotiffDefDir";

```

Comment:
Are GeoTiffs relevant to Simple Geometries?

Reply:
leaving in pending MAP-83